### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by jssol**
### Project requirements

- Made the final design match Zeplin's design.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
```
Comment (medaminedev66) [inline README.md:2]: - [x] Kindly add a description of your project to your Readme so other users can know what the project is about. The project description will give visitors a brief understanding of your project.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @jssol, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your design is great 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (jssol) [reply README.md:2]: @medaminedev66, Thanks for reminding me :pray:
I have added the requested changes.

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @jssol , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**